src/rtp/mtime.rs

```diff
@@ -1,36 +1,127 @@
 #![allow(missing_docs)]
 
 use std::cmp::Ordering;
-use std::ops::{Add, Sub};
+use std::fmt::Display;
```

Comment:
This is not your problem, really, but it's expanding with the increased amount of code in this file.

Could you remove the `#![allow(missing_docs)]` at the top and document the new and old code?

Also, In the `Frequency` doc on `struct`, and `MediaTime` struct, I'd like to see some code examples how to get the `Frequency` for a certain `Format` from the chain `Rtc -> CodecConfig -> PayloadParam -> CodecSpec -> Format -> Frequency` and how to subsequenty use that to construct a `MediaTime`. You can duplicate the code example.

Reply:
I'm not sure what you have in mind for this example so I've not guessed, sorry.